### PR TITLE
🚧 figure_kind_randomizer.py random()

### PR DESCRIPTION
**Pull request by Serg-Bartosh**

### CODE REVIEW

File: src/figure_kind_randomizer.py [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
 
 
 class FigureKindRandomizer:
-    def get_random(self) -> FigureKind:
-        # TODO(AB): Task for Serg
-        pass
+    def __init__(self) -> None:
+        self.__kinds = list(FigureKind)
+
+    def random(self) -> FigureKind:
```
Comment (alena-bartosh) [inline src/figure_kind_randomizer.py]:
```suggestion
    def get_random(self) -> FigureKind:
```

**Review by alena-bartosh — COMMENTED** [review]
Maybe we need make `get_random()` as `@staticmethod`? And do not use `__init__`

File: src/figure_kind_randomizer.py (hunk 1 above)
Comment (s373r) [reply src/figure_kind_randomizer.py]: agree, that is a good naming

**Comment by s373r** [comment]
@Serg-Bartosh , please check @alena-bartosh 's comment above

**Review by alena-bartosh — APPROVED** [review]

**Review by s373r — APPROVED** [review]

**Pull request merged**